test/__snapshots__/TypeScriptGenerator-test.ts.snap

```diff
@@ -85,7 +81,7 @@ export type FeedbackComments_feedback = {
             readonly hasNextPage: boolean | null;
         } | null;
     } | null;
-    readonly " $refType": FeedbackComments_feedback$ref;
+    readonly " $refType": "FeedbackComments_feedback$ref";
```

Comment:
This value doesn’t match the one defined above, which I assume it should? `FragmentRefs<"FeedbackComments_feedback">`

Reply:
Depending on how we do the changes to the DT types this could be fine I suppose?